src/rocker/templates/bashrc_extensions_user_snippet.Dockerfile.em

```diff
@@ -0,0 +1,6 @@
+WORKDIR @home_dir
+RUN echo "\n# Source custom bashrc extensions" >> @home_dir/.bashrc
+@[for path, filename in bashrc_extension_files.items()]
```

Comment:
This is potentially adding a bunch of layers. It can at least collapse the RUN command into only one instance not one per file.